### PR TITLE
minor code fixes for dagster uni dbt course

### PR DESCRIPTION
**Pull request by C00ldudeNoonan**
## Changelog

Insert changelog entry or "NOCHANGELOG" here.

- [ ] `NEW` _(added new feature or capability)_
- [ ] `BUGFIX` _(fixed a bug)_
- [ ] `DOCS` _(added or updated documentation)_

### CODE REVIEW

File: docs/dagster-university/pages/dagster-dbt/lesson-5/3-creating-assets-that-depend-on-dbt-models.md [hunk 1]
```diff
@@ -107,53 +107,52 @@ Now we’re ready to create the asset!
...
-                "trips": "Number of Trips",
-                "destination_borough": "Destination Borough"
-            },
-        )
-
-        pio.write_image(fig, constants.AIRPORT_TRIPS_FILE_PATH)
-
-        with open(constants.AIRPORT_TRIPS_FILE_PATH, 'rb') as file:
-            image_data = file.read()
+   @asset(
```
Comment (cmpadden) [inline docs/dagster-university/pages/dagster-dbt/lesson-5/3-creating-assets-that-depend-on-dbt-models.md:110]: What is the difference in this block?

Comment (C00ldudeNoonan) [reply docs/dagster-university/pages/dagster-dbt/lesson-5/3-creating-assets-that-depend-on-dbt-models.md:110]: the code block has a leading space which is a little tedious to copy over.

Comment (cmpadden) [reply docs/dagster-university/pages/dagster-dbt/lesson-5/3-creating-assets-that-depend-on-dbt-models.md:110]: ty ty

**Review by cmpadden — APPROVED** [review]
Will just need to update `project-dagster-university` to use the `resource_type =` change.

**Pull request merged**